include/nighthawk/client/options.h

```diff
@@ -40,6 +40,7 @@ class Options {
   virtual uint32_t maxPendingRequests() const PURE;
   virtual uint32_t maxActiveRequests() const PURE;
   virtual uint32_t maxRequestsPerConnection() const PURE;
+  virtual std::string sequencerIdleStrategy() const PURE;
```

Comment:
Same here, I think it'd be nicer to have an `enum class` to represent the legal values rather than a string.